lib/common/netcdf-check.sh

```diff
@@ -91,6 +91,7 @@ export -f _get_checker_code_dir
 # "$@" - checker suites passed
 add_checker_signature() {
     local file=$1; shift
+    local checks=$@
```

Comment:
how do you expect checks to be equal to 'cf, imos' ? since  add_checker_signature always used to take only a file as an argument ? Are you planing to modify the generic incoming handler as well ?